server/bin/pbench-server-activate-create-results-dir-structure

```diff
@@ -90,14 +60,19 @@ typeset -i nerrs=0
 
 # it's OK if the directories exist already.
 mkdir -p ${archive_dir}
+
+pbench_incoming_dir=${pbench_dir}/public_html/incoming
```

Comment:
We seem to use `pbench_` prefixes to denote variables pulled directly from the config file.  Do we want to keep this nomenclature?

Reply:
I think that's too much trouble for this script: it already uses the pbench_ prefixed names and changing all of them is just a source of errors that I would rather avoid. Besides, this script will be going away after PR#1099 is merged.

Reply:
Not sure, but it seems these variables still exist in #1199.

Reply:
I don't think so. I just did a global search for them and did not find them. I'm not sure whether I did it correctly, but I was trying to follow your instructions about making #1199 dependent on this PR. The commits in #1199 are a superset of the commits in this PR. So maybe you are looking at the "common" commits?

Reply:
> So maybe you are looking at the "common" commits?

I see now that the 3 commits in this PR are the exact same commits in PR #1199.  I think what through me was:

> Besides, this script will be going away after PR #1199 is merged.

I read that to mean that PR #1199 will remove these, but it doesn't.

I'll drop this asking for this change, but really hope we won't get bit by the mixing nomenclature since this is going to live for longer than expected.